### PR TITLE
Feature/update graphql

### PR DESCRIPTION
**Pull request by leticiawanderley**
#### What is the purpose of this pull request?
* Update graphql queries and mutations context.
* Improve store layout.

#### What problem is this solving?
The checkout queries and mutations needed to be private in order to send `cookies` to the server. 

#### Types of changes
* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Banner.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ class Banner extends Component {
   render() {
     const { image, page, targetParams } = this.props
     return (
-      <div>
-        <Link page={page} params={targetParams}>
+      <div className="flex items-center">
+        <Link className="mb0" page={page} params={targetParams}>
           <img src={image} />
```
Comment (kevinch) [inline react/Banner.js]: `img` should have `alt` attribute for a11y. Can you add it in this case?

File: react/ProductPage.js [hunk 2]
```diff
@@ -13,23 +13,31 @@ class ProductPage extends Component {
...
     const { data } = this.props
     const { product, loading } = data
-    if (loading) {
-      return null
-    }
+
     return (
       <div>
         {loading && <WrappedSpinner />}
         {!loading && (
```
Comment (kevinch) [inline react/ProductPage.js]: Ternary could be easier to read in this case: 
```
{loading 
    ? <WrappedSpinner />
    : <div className="flex flex-column items-center justify-center pv6 pv9-ns">
```

File: react/components/Footer.js [hunk 3]
```diff
@@ -3,9 +3,14 @@ import React, { Component } from 'react'
 export default class Footer extends Component {
   render() {
     return (
-      <footer className="fixed w-100 bottom-0 pa4 pa5-l bg-light-gray">
+      <footer className="w-100 bottom-0 pa7 pa9-l bg-near-white flex items-center">
         <div className="mw8 center">
-          <div className="mb4-l cf" />
+          <div className="cf tc fw7 flex items-center justify-center">
+            POWERED BY VTEX
```
Comment (kevinch) [inline react/components/Footer.js]: Hard coded i18n?

File: react/components/Footer.js [hunk 4]
```diff
@@ -3,9 +3,14 @@ import React, { Component } from 'react'
...
+          </div>
+          <div className="tc">
+            2018
```
Comment (kevinch) [inline react/components/Footer.js]: Hard coded date?

File: react/components/Header.js [hunk 5]
```diff
@@ -25,31 +27,31 @@ class Header extends Component {
...
+  handleCart = () => location.assign('/checkout/#/cart')
 
   render() {
     const { name } = this.props
     const { searchValue } = this.state
     return (
-      <div className="flex-ns justify-between items-center w-100 top-0 pa2 pa5-l bg-light-gray">
-        <a className="link b f3 near-black mt3" href="/">
+      <div className="z-2 flex-ns justify-between items-center w-100 top-0 pa4 pa5-ns bg-white bb bw1 b--serious-black tc tl-ns">
+        <a className="link b f3 near-black tc tl-ns" href="/">
```
Comment (kevinch) [inline react/components/Header.js]: What is this `a` tag doing?

File: react/components/ShelfItem.js [hunk 6]
```diff
@@ -18,21 +20,23 @@ export default class ShelfItem extends Component {
...
-          <img src={imageUrl} width="200" height="200" />
-          <span>{name}</span>
-          <FormattedNumber
-            value={price}
-            style="currency"
-            currency={this.context.culture.currency}
-            minimumFractionDigits={2}
-            maximumFractionDigits={2}
-          />
+          <img src={imageUrl} />
```
Comment (kevinch) [inline react/components/ShelfItem.js]: `alt` attribute would help a11y.

File: react/images/CartIcon.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+          <path d='M20,17H6c-0.501,0-0.925-0.371-0.991-0.868L3.125,2H0V0h4c0.501,0,0.925,0.371,0.991,0.868L5.542,5H23 c0.316,0,0.614,0.149,0.802,0.403c0.189,0.254,0.247,0.582,0.156,0.884l-3,10C20.831,16.71,20.441,17,20,17z'
+          />
+      </g>
+    </svg>
+    )
+  }
+}
+
+CartIcon.propTypes = {
+  fill: PropTypes.string,
```
Comment (kevinch) [inline react/images/CartIcon.js]: Prop is not used.

File: react/slider.css [hunk 8]
```diff
@@ -209,7 +210,7 @@
 /* Dots */
 
 .slick-slider {
-  margin-bottom: 30px;
+  margin-bottom: 0px;
```
Comment (kevinch) [inline react/slider.css]: Hint: when set to 0, css value can omit the unit.

File: react/Banner.js (hunk 1 above)
Comment (klzns) [reply react/Banner.js]: Bem lembrado. Tem que ter que adicionar a settings da imagem pra receber não so a URL como o texto de `alt`.

File: react/ProductPage.js [hunk 9]
```diff
@@ -13,23 +13,31 @@ class ProductPage extends Component {
     data: PropTypes.object,
   }
 
+  static contextTypes = {
```
Comment (klzns) [inline react/ProductPage.js]: Criar um HOC pra passar essa função.

Principalmente agora que refizeram context no React mais recente.

File: react/Shelf.js [hunk 10]
```diff
@@ -25,28 +30,38 @@ class Shelf extends Component {
...
+        title: 'From',
+        type: 'number',
       },
     },
   }
 
-  render() {
-    const { data } = this.props
+  trimName = name => {
+    return name.length > 20 ? `${name.substring(0, 18)}...` : name
```
Comment (klzns) [inline react/Shelf.js]: @kevinch como deixou passar essa elipsis?

Comment (klzns) [reply react/Shelf.js]: Usar `&hellip;`

File: react/components/Header.js (hunk 5 above)
Comment (klzns) [reply react/components/Header.js]: Linkando pra home

**Pull request merged**